src/edwards/curve.rs

```diff
@@ -120,6 +125,11 @@ impl EllipticCurve for Curve {
             z: self.f.one()
         }))
     }
+    fn from_bytes_be(&self, buf: &[u8]) -> Result<Self::Deser,std::io::Error> {
```

Comment:
In the three models, it doesn't make sense to have le/be serializers for points.
Scalars and field elements can be written in be/le order, but points not.
So, you can create a special trait for Points, something like Encode/Decode, or using FromFactory trait.

Reply:
In case of encoding, it would be good to receive as input whether the encoding is compressed or not. 
Edwards points are always compressed, hence the encode method discards its input.

Reply:
This makes sense, I've split up the traits now.